packages/loader/container-loader/src/container.ts

```diff
@@ -1385,6 +1397,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         if (value === ConnectionState.Connected) {
             this._clientId = this.pendingClientId;
+            if (this._clientId !== undefined) {
```

Comment:
It is never undefined here.

Reply:
The linter got mad at me for using the non-null assertion.  Should have just added the ignore I guess 🙃